openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/CommonApiConfiguration.java

```diff
@@ -60,6 +60,15 @@ public class CommonApiConfiguration {
 	@Parameter
 	protected Boolean loopbackOperationName;
 
+	@Parameter
+	protected Boolean defaultNonNullableFields;
+
+	@Parameter
+	protected String nonNullableAnnotation;
+
+	@Parameter
```

Comment:
All fields in the CommonApiConfiguration can be overridden in the ApiConfiguration. Could you complete the "mergeWithCommonApiConfiguration" with your new fields?